src/components/MovieView/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import BlockMovieView from "components/BlockMovieView";
+import ListMovieView from "components/ListMovieView";
+
+export default function MovieView(props) {
+  const [movieInfo, isActive] = [props.movieInfo, props.isActive];
```

Comment:
const {movieInfo, isActive} = props
https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0